### DISCUSSION TITLE
Curve filling issue

### DISCUSSION THREAD

**Discussion by mike9755** [pyqtgraph/pyqtgraph#2255] [General]
Hi Dears

Please help me if is possible to fill the plot from a defined Y value. shows as below picture, mean filling area start from y=1 or similar, thank you.
![image](https://user-images.githubusercontent.com/5673534/163209192-fd4bd008-02ed-4f92-84cb-899bd2bdd85f.png)

Test code as below:

```python
import numpy as np
import pyqtgraph as pg
from pyqtgraph.Qt import QtCore

app = pg.mkQApp("Plotting Example")

win = pg.GraphicsLayoutWidget(show=True, title="Basic plotting examples")
win.resize(1000,600)
win.setWindowTitle('pyqtgraph example: Plotting')
pg.setConfigOptions(antialias=True)

p7 = win.addPlot(title="Filled plot, axis disabled")
y = np.sin(np.linspace(0, 10, 1000)) + np.random.normal(size=1000, scale=0.1)

y+=3

p7.plot(y, fillLevel=-0.3, brush='y',pen='y')
p7.showAxis('bottom', False)

if __name__ == '__main__':
    pg.exec()
```

**Comment by ixjlyons** [comment]
This example sets `fillLevel=-0.3`, so you can change that to `fillLevel=1` to fill from y = 1 up.

There's also [`FillBetweenItem`](https://pyqtgraph.readthedocs.io/en/latest/graphicsItems/fillbetweenitem.html) to fill between two arbitrary curves instead of a constant value.

**Reply by mike9755** [reply]
This is exactly what I need, thank you so much.
